src/components/LoanCards/KvClassicLoanCardContainer.vue

```diff
@@ -471,6 +472,7 @@ export default {
 			const addedLoan = {
 				id: this.loan?.id,
 				name: this.loan?.name ?? '',
+				gender: this.loan?.gender ?? '',
```

Comment:
We are also adding this on the BP lend CTA component.
https://github.com/kiva/ui/blob/0305b05ac8c0585eada6b23f1df5240782254341/src/components/BorrowerProfile/LendCta.vue#L674